src/content-helper/common/components/tone-selector/component.tsx

```diff
@@ -72,13 +69,13 @@ type FixedToneProp = keyof typeof PARSELY_TONES;
 const TONE_LIST = Object.keys( PARSELY_TONES ) as ToneProp[];
 
 /**
- * Returns the label for a given tone.
+ * Returns the value for a given tone.
  *
  * @since 3.13.0
  *
- * @param {ToneProp} tone The tone to get the label for.
+ * @param {ToneProp} tone The tone to get the value for.
  *
- * @return {string} The label for the given tone.
+ * @return {string} The value for the given tone.
```

Comment:
Same thing I reported in a previous file about value and label. Ignore if this was done on purpose.